linux_os/guide/services/ntp/package_ntp_installed/rule.yml

```diff
@@ -11,7 +11,7 @@ severity: high
 references:
     anssi: NT012(R03)
     disa: "160"
-    nist: AU-8(1)
+    nist: AU-8(1),AU-12(1)
```

Comment:
```suggestion
    nist: CM-6(a),CM-6(b)
```

Reply:
This one is hard. AU-8(1)(a) and AU-8(1)(b) need a time server, but the NIST controls are not specific enough to map to installing packages.

What if we use CM-6(a) and CM-6(b), which is kind of a catch-all "create and implement configuration settings" requirement?

Reply:
Updating to CM is out of scope of this PR.